docs/Learning-Environment-Create-New.md

```diff
@@ -529,28 +491,73 @@ the Python API work as expected using the `notebooks/getting-started.ipynb`
+RollerAgent takes about 300,000 steps to train. However, you can change the 
+following hyperparameters  to speed up training considerably (to under 20,000 steps):
+
+    batch_size: 1
+    buffer_size: 10
+
+Since this example creates a very simple training environment with only a few inputs 
+and outputs, using small batch and buffer sizes speeds up the training considerably. 
+However, if you add more complexity to the environment or change the reward or 
+observation functions, you might also find that training performs better with different 
+hyperparameter values.
+
+**Note:** In addition to setting these hyperparameter values, the Agent 
+**DecisionFrequency** parameter has a large effect on training time and success.
+A larger value reduces the number of decisions the training algorithm has to consider and,
+in this simple environment, speeds up training. 
+
+To train in the editor, run the following Python command from a Terminal or Console 
+window before pressing play:
+
+    mlagents-learn config/config.yaml --run-id=RollerBall-1 --train
+
+(where `config.yaml` is a copy of `trainer_config.yaml` that you have edited 
+to change the `batch_size` and `buffer_size` hyperparameters for your brain.)
+
+**Note:** If you get a `command not found` error when running this command,  make sure 
+that you have followed the *Install Python and mlagents Package* section of the 
+ML-Agents [Installation](Installation.md) instructions.
+
+To monitor the statistics of Agent performance during training, use 
+[TensorBoard](Using-Tensorboard.md). 
+
+![TensorBoard statistics display](images/mlagents-RollerAgentStats.png)
+
+In particular, the *cumulative_reward* and *value_estimate* statistics show how 
+well the Agent is achieving the task. In this example, the maximum reward an 
+Agent can earn is 1.0, so these statistics approach that value when the Agent
+has successfully *solved* the problem.
+
+**Note:** If you use TensorBoard, always increment or change the `run-id` 
+you pass to the `mlagents-learn` command for each training run. If you use 
+the same id value, the statistics for multiple runs are combined and become 
+difficult to interpret.
+
 ## Review: Scene Layout
 
 This section briefly reviews how to organize your scene when using Agents in
 your Unity environment.
 
 There are two kinds of game objects you need to include in your scene in order
-to use Unity ML-Agents:
-
-* Academy
-* Agents
-
-You also need to have brain assets linked appropriately to Agents and to Academy
+to use Unity ML-Agents: an Academy and one or more Agents. You also need to 
+have brain assets linked appropriately to your Agents and to the Academy.
 
 Keep in mind:
 
 * There can only be one Academy game object in a scene.
-* You can only train Learning Brains that have been included into the Academy's  
-  Broadcast Hub.
+* You can only train Learning Brains that have been added to the Academy's  
+    Broadcast Hub list.
```

Comment:
Why a new line here ?

Reply:
Not sure what you mean. It was there in the original.